core/scripts/server.ts

```diff
@@ -184,7 +184,11 @@ app.get('*', (req, res, next) => {
   }
 
   const site = req.headers['x-vs-store-code'] || 'main'
-  const cacheKey = `page:${site}:${req.url}`
+  const cacheKey = serverHooksExecutors.beforeBuildCacheKey({
```

Comment:
what value has `cacheKey` if `serverHooks.beforeBuildCacheKey(({ req, site }) => {` is commented-out?